fix(public-page-layout): update logos used in MC authentication flows

tylermorrisford opened the pull request:
#### Summary

This PR updates all logos within `packages/assets/logos/` to use the new branding, using the existing names.


#### Description

| PublicPageLayout Before      | PublicPageLayout After |
| ----------- | ----------- |
| ![Screenshot 2024-01-25 at 11 25 16 AM](https://github.com/commercetools/merchant-center-application-kit/assets/49931992/a7a6fcef-dfec-43bb-a271-d4e9c6c78582) |  ![Screenshot 2024-01-25 at 11 35 05 AM](https://github.com/commercetools/merchant-center-application-kit/assets/49931992/9855cd31-72ac-4f56-a54e-2a886c27fad0)|
| Navbar Before      | Navbar After |
|![Screenshot 2024-01-25 at 11 36 38 AM](https://github.com/commercetools/merchant-center-application-kit/assets/49931992/037a8586-1419-4b6d-a98f-73747e01d520) | ![Screenshot 2024-01-25 at 11 39 28 AM](https://github.com/commercetools/merchant-center-application-kit/assets/49931992/4624ff27-6eaa-4242-9f97-534e4fed51b2) |

CarlosCortizasCT commented:
I was expecting us to change all the logos at once to not live with inconsistencies.

We have quite a few other logos [here](https://github.com/commercetools/merchant-center-application-kit/tree/main/packages/assets/logos).

@FilPob What do you think about it?

lidiyacommercetools commented:
> I was expecting us to change all the logos at once to not live with inconsistencies.
> 
> We have quite a few other logos [here](https://github.com/commercetools/merchant-center-application-kit/tree/main/packages/assets/logos).
> 
> @FilPob What do you think about it?

The only reason we decided to switch this particular logo is because we were specifically asked to by marketing earlier today. However, if this is problematic for a future implementation, we can discuss.

FilPob commented:
@Shecki  are there any plans about when to replace all the logos in the MC? I think ideally we would do that in one go as Carlos suggested
@CarlosCortizasCT if we replace the files, can you send us links to the pages where these logos are used for reviewing?

CarlosCortizasCT commented:
> @Shecki are there any plans about when to replace all the logos in the MC? I think ideally we would do that in one go as Carlos suggested @CarlosCortizasCT if we replace the files, can you send us links to the pages where these logos are used for reviewing?

@FilPob I'm not sure about it because in this PR we only change the logo in the `PublicPageLayout` component, which is exported and might be used from any consumer application.

I took a look at the Merchant Center monorepo and I only found one usage (in the login page).

![image](https://github.com/commercetools/merchant-center-application-kit/assets/97907068/19ef1802-a2e9-4074-8fba-27ec8e15bbb4)

lidiyacommercetools commented:
@FilPob and @CarlosCortizasCT , to circle back, is this update of the image problematic? It was requested by marketing but the goal is to make the rebranding as unproblematic for Shield as possible. Will updating the one image be an issue for your team down the road?

FilPob commented:
@lidiyacommercetools am coming back with the update from Sven :) He agreed that we should update all logos in MC at once. (no need to wait for the new colors) I can provide the updated logos this week... Not sure though what does that mean for this PR. Wdyt @CarlosCortizasCT ? should we merge it and update the other logos in a separate one or what?

CarlosCortizasCT commented:
> @lidiyacommercetools am coming back with the update from Sven :) He agreed that we should update all logos in MC at once. (no need to wait for the new colors) I can provide the updated logos this week... Not sure though what does that mean for this PR. Wdyt @CarlosCortizasCT ? should we merge it and update the other logos in a separate one or what?

I think it would be better if we update all logos in the same PR.

lidiyacommercetools commented:
@CarlosCortizasCT , that's fine. I will let marketing know.

lidiyacommercetools commented:
@CarlosCortizasCT , to confirm, we will cancel this PR and I will inform marketing. And Shield will update all of the logos in the near future. Correct?

CarlosCortizasCT commented:
> @CarlosCortizasCT , to confirm, we will cancel this PR and I will inform marketing. And Shield will update all of the logos in the near future. Correct?

@lidiyacommercetools we can do whatever is more convenient.

From my point of view, we can keep this PR open and, when @FilPob  provides the rest of the logos, he will share a link to them and Tyler can update this PR with them.

Would that work for you?

lidiyacommercetools commented:
@CarlosCortizasCT , sounds like a plan. We will put this PR on hold until the other logos are ready to go.

tylermorrisford commented:
Thank you @CarlosCortizasCT and @FilPob! @lidiyacommercetools I added the on hold label here.

FilPob commented:
Here we go. all the updated logos [HERE](https://drive.google.com/drive/folders/1I6xpkO7xeZP4DFf1-DWaAjxq4NbPtN-p?usp=sharing) :)

tylermorrisford commented:
Ok! @FilPob @CarlosCortizasCT @lidiyacommercetools All logos updated. I'm happy to find where they are used to and take screenshots in the visual testing app but may need some guidance as I'm not familiar with where all of these logos are used. Regardless thank you all for your help!

CarlosCortizasCT reviewed (approved):
This looks good to me now 💯 

Thanks a lot to everyone involved for your patience 🙇

CarlosCortizasCT commented:
> Ok! @FilPob @CarlosCortizasCT @lidiyacommercetools All logos updated. I'm happy to find where they are used to and take screenshots in the visual testing app but may need some guidance as I'm not familiar with where all of these logos are used. Regardless thank you all for your help!

Regarding the Visual Testing Application, the changes can already be seen in [this Percy check](https://percy.io/commercetools-GmbH/app-kit-components/builds/32293463?utm_campaign=commercetools-GmbH&utm_content=app-kit-components&utm_source=github_status_public).

If @FilPob needs anything else, I think we can search for usages of this logos in MC frontend applications.

FilPob commented:
Yeah it would be great to see some screenshots if everything looks as expected

tylermorrisford commented:
> Yeah it would be great to see some screenshots if everything looks as expected

@FilPob I updated the description here 👆  with what I've found so far; it looks like the `application loader` image is loaded from assets/images/ instead of assets/logos/ - maybe we should update that in this PR as well? @CarlosCortizasCT WDYT?

emmenko commented:
Thanks for the collaboration!

Two things are still missing:
- The changeset
- We also have an embedded svg logo on the index html template

CarlosCortizasCT commented:
> > Yeah it would be great to see some screenshots if everything looks as expected
> 
> @FilPob I updated the description here 👆 with what I've found so far; it looks like the `application loader` image is loaded from assets/images/ instead of assets/logos/ - maybe we should update that in this PR as well? @CarlosCortizasCT WDYT?
> 
> Other than these two, I'm struggling to find where the rest of the logos are used.

I think we need to also have new logos for the ones we have in the `images` folder and, after we have them, I agree it would maybe make sense to move the to the `logos` directory.
@FilPob you can find the three SVGs we would need to have updated in [this directory](https://github.com/commercetools/merchant-center-application-kit/tree/main/packages/assets/images):
* [ct-logo.svg](https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/assets/images/ct-logo.svg)
* [ct-logo_bw.svg](https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/assets/images/ct_logo_bw.svg)
* [ct-logo_bw_white.svg](https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/assets/images/ct_logo_bw_white.svg)

Regarding where the logos are used, I don't know neither.
I took a look at the MC frontend and couldn't find any usage.

FilPob commented:
Sure, here they are: 
![ct_logo_bw_white](https://github.com/commercetools/merchant-center-application-kit/assets/94548534/4cf6e235-57c8-4536-8ba4-88ffa30f041c)
![ct_logo_bw](https://github.com/commercetools/merchant-center-application-kit/assets/94548534/d66eb84f-56af-4870-9c79-ce3afb7981d7)
![ct-logo](https://github.com/commercetools/merchant-center-application-kit/assets/94548534/cf971db6-a6e2-4a89-98f5-42baaec0fefc)

FilPob commented:
Sure, here they are: 
![ct_logo_bw_white](https://github.com/commercetools/merchant-center-application-kit/assets/94548534/24242cbf-3cc7-4b4a-b39f-c0c96e84bb10)
![ct_logo_bw](https://github.com/commercetools/merchant-center-application-kit/assets/94548534/59d7feec-34d2-4d9c-94f6-cbae091bc43f)
![ct-logo](https://github.com/commercetools/merchant-center-application-kit/assets/94548534/76ee173b-eca5-499a-9365-ac43c4ddc25e)

FilPob commented:
![ct_logo_bw_white](https://github.com/commercetools/merchant-center-application-kit/assets/94548534/6198c413-edc4-4756-a457-6da3a8554f5e)
![ct_logo_bw](https://github.com/commercetools/merchant-center-application-kit/assets/94548534/792d2856-2154-4280-9c9a-e93df671cf7e)
![ct-logo](https://github.com/commercetools/merchant-center-application-kit/assets/94548534/f615af29-e017-47b8-8445-4b98d0cb24f2)

FilPob commented:
sure here are the additional ones: https://drive.google.com/drive/folders/1JADQZ1pPwkbv-0l4MsBLdDC52UZt2OqC?usp=sharing

tylermorrisford commented:
> Thanks for the collaboration!
> 
> Two things are still missing:
> 
> * The changeset
> * We also have an embedded svg logo on the index html template (`packages/mc-html-template/html-docs/application.html`)

Thank you @emmenko! Do we know which file should replace the embedded image in  `application.html` at line 50?

tdeekens commented:
Should be here. You need to hand embed it. It's inlined for quick loading. I think we changed it 5 years ago the last time 🤗. 

Let us known if you need a guiding hand on Monday. It surely needs a release of app-kit. 

https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/mc-html-template/html-docs/application.html

tylermorrisford commented:
> Should be here. You need to hand embed it. It's inlined for quick loading. I think we changed it 5 years ago the last time 🤗.
> 
> Let us known if you need a guiding hand on Monday. It surely needs a release of app-kit.
> 
> https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/mc-html-template/html-docs/application.html

Beauty - thank you @tdeekens! Updated in [6bf1e8b](https://github.com/commercetools/merchant-center-application-kit/pull/3379/commits/6bf1e8bac06c0ed31c61c906f444f72510a01e41)

tylermorrisford commented:
Is it possible to get a screenshot of the `application.html` change without deploying? As far as I can tell the static assets are loaded from their environments. This is what I see if I load the updated `application.html` locally:
![Screenshot 2024-01-29 at 8 03 24 AM](https://github.com/commercetools/merchant-center-application-kit/assets/49931992/468d1157-28f1-40f5-8ebb-57d9df252bcf)

stephsprinkle commented:
@commercetools/shield-team-design-system @FilPob I believe this PR takes care of updating all the logos in app kit and is ready for your final review.

CarlosCortizasCT commented on packages/mc-html-template/html-docs/application.html:
```diff
@@ -1,35 +1,65 @@
+    <meta name="referrer" content="no-referrer" />
 
-    <meta http-equiv="Content-Security-Policy" content="__CSP__">
+    <meta http-equiv="Content-Security-Policy" content="__CSP__" />
 
     <!-- Preconnects -->
-    <link rel="preconnect" href="__CDN_URL__">
-    <link rel="preconnect" href="__MC_API_URL__">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link rel="preconnect" href="https://app.launchdarkly.com">
-    <link rel="preconnect" href="https://clientstream.launchdarkly.com">
-    <link rel="preconnect" href="https://events.launchdarkly.com">
```
I'm wondering about all this style changes 🤔 

I believe we should only be applying the expected inline svg changes.

tylermorrisford replied:
This change was made by Prettier - should I ignore Prettier for now?

CarlosCortizasCT replied:
Yes, that would be great. Please 🙏

CarlosCortizasCT commented:
Since @emmenko also [requested some changes](https://github.com/commercetools/merchant-center-application-kit/pull/3379#issuecomment-1911530303) (already applied) I would like to wait for his approval too.

emmenko reviewed (approved):
Thanks! 💯
